Author: Philipp Benner <mail@philipp-benner.de>
Description: Replace isnumber() (which does not exist on linux) by isdigit().

--- wise-2.4.1.orig/src/models/phasemodel.c
+++ wise-2.4.1/src/models/phasemodel.c
@@ -20,7 +20,7 @@
     if( line[0] == '#' ) {
       continue;
     }
-    if( !isnumber(line[0]) ) {
+    if( !isdigit(line[0]) ) {
       warn("Bad looking line in intron file, %s",line);
       continue;
     }
